src/WebJobs.Script.WebHost/WebScriptHostManager.cs

```diff
@@ -433,22 +433,18 @@ protected override void OnInitializeConfig(ScriptHostConfiguration config)
             // Add our WebHost specific services
             var hostConfig = config.HostConfig;
             hostConfig.AddService<IMetricsLogger>(_metricsLogger);
-
-            config.HostConfig.AddService<IWebHookProvider>(this._bindingWebHookProvider);
```

Comment:
Just cleaning up this code a bit. Also removing the explicit use of AmbientConnectionStringProvider, since config.DashboardConnectionString getter does the same.